test/dlc_tests/container_tests/bin/benchmark/run_pytorch_training_performance_gpu_imagenet

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+PYTHON_VERSION=$(python -c 'import sys; print(sys.version_info[0])' | tr -d "'")
+if [ "$PYTHON_VERSION" -eq 2 ]
+then
+  exit 0
+fi
+TIMESTAMP=$(date "+%Y-%m-%d-%H-%M-%S")
+HOME_DIR=/test/benchmark
+BIN_DIR=${HOME_DIR}/bin
+LOG_DIR=${HOME_DIR}/logs/gpu
+LOG_FILE=image_results_${COMMIT_INFO}_${TIMESTAMP}.txt
+
+mkdir -p ${HOME_DIR}
+mkdir -p ${BIN_DIR}
+mkdir -p ${LOG_DIR}
+
+cd ~
+echo Benchmark Preparation started: >&2
+START=$(date +%s)
+timeout 5m python examples/imagenet/main.py \
+       -a resnet50 \
+       --dist-url 'tcp://127.0.0.1:8080' \
+       --dist-backend 'nccl' \
+       --multiprocessing-distributed \
+       --world-size 1 \
+       --rank 0 \
+       --workers 30 \
+       --epochs 0 \
+       ~/imagenet
+END=$(date +%s)
+DIFF=$(( END - START ))
+echo Preparation Stage took $(( DIFF/60 )) mins to finish>&2
+
+echo Benchmark started: >&2
+START=$(date +%s)
+timeout 60m python examples/imagenet/main.py \
+       -a resnet50 \
+       --dist-url 'tcp://127.0.0.1:8080' \
+       --dist-backend 'nccl' \
+       --multiprocessing-distributed \
+       --world-size 1 \
+       --rank 0 \
+       --workers 30 \
+       --epochs 16 \
+       ~/imagenet 2>&1 | tee ${LOG_DIR}/${LOG_FILE}
+python /test/bin/benchmark/get_pytorch_performance_num.py ${LOG_DIR}/"${LOG_FILE}" >> ${LOG_DIR}/"${LOG_FILE}"
```

Comment:
Is there anywhere where we define a failure condition or threshold for this test?

Reply:
Will address it separately